.zuul.yaml

```diff
@@ -0,0 +1,28 @@
+- project:
+    name: kubernetes-sigs/cluster-api-provider-openstack
+    check:
+      jobs:
+        - cluster-api-provider-openstack-conformance-stable
+
+- job:
+    name: cluster-api-provider-openstack-conformance-stable
+    parent: init-test
+    timeout: 36000
+    description: |
+      Test CAPO & Kubernetes master against CAPI & CABPKG stable
+    run: .zuul/playbooks/run.yaml
+    nodeset: ubuntu-xenial-citynetwork
```

Comment:
devstack now only supports ubuntu18.04. Use `ubuntu-bionic` instead.

Reply:
I assume without citynetwork suffix?

Reply:
yeah, if you don't use citynetwork cloud API
otherwise, please wait for this PR: https://github.com/theopenlab/openlab-zuul-jobs/pull/776